code/_onclick/click_override.dm

```diff
@@ -81,7 +81,12 @@
 				L.adjustStaminaLoss(60)
 				L.Jitter(10 SECONDS)
 				var/atom/throw_target = get_edge_target_turf(user, get_dir(user, get_step_away(L, user)))
-				L.throw_at(throw_target, powergrid / 100000, powergrid / 100000) //100 kW in grid throws 1 tile, 200 throws 2, etc.
+				if(ishuman(L))
+					var/mob/living/carbon/human/H = L
+					if(H.gloves && H.gloves.siemens_coefficient == 0) //No throwing with insulated gloves (you still get stamina however)
+						break
+				L.throw_at(throw_target, powergrid / 150000, powergrid / 300000) //150 kW in grid throws 1 tile, 300 throws 2, etc.
```

Comment:
Could these be converted to `150/300 KW` maybe? We do actually have a define for this, it just sits in `pulsedemon_abilities.dm` for some reason

Reply:
Done!